tools/upgrade.sh

```diff
@@ -34,7 +33,7 @@ then
   printf '%s\n' '                        /____/                            '
   printf "${BLUE}%s\n" "Hooray! Oh My Bash has been updated and/or is at the current version."
   printf "${BLUE}${BOLD}%s${NORMAL}\n" "To keep up on the latest news and updates, follow us on GitHub: https://github.com/ohmybash/oh-my-bash"
-  exec bash; source $HOME/.bashrc
+  exec bash; source ~/.bashrc
```

Comment:
@akinomyoga We can change like this to replace the current shell instead of adding a new shell on top of the current.

```suggestion
  exec -l bash;
```

Reply:
> We can change like this to replace the current shell instead of adding a new shell on top of the current.

`exec bash` already does it.

- Is there any reason for marking the replaced process as the login shell by `-l`?
- `source ~/.bashrc` is a fallback code in case that `bash` is not found in `PATH` and `exec bash` fails.

Reply:
Since this is actually unrelated to the change of this PR, I now merge this PR. I address this point in another PR #292.